lms/djangoapps/learner_home/mock_data.json

```diff
@@ -225,50 +222,50 @@
                 "isEmailEnabled": false,
                 "hasOptedOutOfEmail": false,
                 "isEnrolled": true,
-                "isVerified": false
+                "isVerified": false,
+                "lastEnrolled": null
             },
-            "certificates": {
+            "certificate": {
                 "availableDate": null,
                 "isRestricted": false,
                 "isEarned": false,
                 "isDownloadable": false,
                 "certPreviewUrl": "edx.com/courses/my-course-url/cert-preview"
             },
-            "provider": {
-                "name": "edX Course Provider",
-                "website": "www.edx.com",
-                "email": "support@edx.com"
+            "courseProvider": {
+                "name": "edX Course Provider"
             },
             "course": {
-                "title": "Course Name 3",
-                "bannerUrl": "https://edx-cdn.org/v3/prod/logo.svg",
+                "courseName": "Course Name 3",
+                "bannerImgSrc": "https://edx-cdn.org/v3/prod/logo.svg",
```

Comment:
...why bannerImgSrc when programs banner is still bannerUrl?

Reply:
We explicitly changed in in course, but didn't get a chance to explicitly change in programs. That was one of my open questions: do we want to update it in programs & suggested courses?